Update deployment methods

Red-GV opened the pull request:
This PR updates the `Makefile` and `run.sh` to have new deployment methods for each type of Loki deployment.

periklis commented on run.sh:
```diff
@@ -2,39 +2,259 @@
+    $GINKGO --output-dir=$output_directory --json-report=report.json --junit-report=report.xml --timeout=4h ./benchmarks
+}
+
+enable_ocp_prometheus_monitoring() {
+    echo -e "\nAdding user workload monitoring configuration"
+
+    kubectl -n openshift-monitoring apply -f $ocp_prometheus_config_path/cluster-monitoring-config.yaml
+	kubectl -n openshift-user-workload-monitoring apply -f $ocp_prometheus_config_path/user-workload-monitoring-config.yaml
+}
+
+disable_ocp_prometheus_monitoring() {
+    echo -e "\nRemoving user workload monitoring configuration"
+
+    kubectl -n openshift-monitoring delete -f $ocp_prometheus_config_path/cluster-monitoring-config.yaml --ignore-not-found=true
+	kubectl -n openshift-user-workload-monitoring delete  -f $ocp_prometheus_config_path/user-workload-monitoring-config.yaml --ignore-not-found=true
+}
```
Suggestion to call it with it's real name: `(enable|disable)_ocp_user_workload_monitoring`

periklis reviewed (commented):
Looks good to me, a couple of suggestions

Red-GV commented:
@periklis I have addressed your comments. The `build` test is currently failing because I have updated the benchmarks to work with the latest changes in `cluster-logging-load-client`, which needs to be rebuilt.
